lib/portlayer/exec/handle.go

```diff
@@ -283,6 +284,19 @@ func Create(ctx context.Context, sess *session.Session, config *ContainerCreateC
 		Metadata: config.Metadata,
 	}
 
+	dsType, err := sess.Datastore.Type(ctx)
```

Comment:
@hickeng What do you think? You suggested we added the vsan logic here in handle.go.